### ISSUE TITLE
Hot water cylinder energy use

### ISSUE THREAD

**Issue by TrystanLea** [emoncms/MyHomeEnergyPlanner#359]
I noticed when adding a hot water cylinder e.g:

_HWS23: Indirectly heated cylinder, up to 130 litres, 38mm factory fitted foam insulation_

Hot water demand increased significantly from:

    Hot water energy requirement 1383.95 kWh/year
    Space heating energy requirement 17071.01 kWh/year

to:

    Hot water energy requirement 6018.94 kWh/year
    Space heating energy requirement 14889.16 kWh/year

6018 kWh seems a bit high?

**Comment by marianneURBED** [comment]
Not an issue I’ve come across – don’t think I’ve used that particular set up ever - but that could be an error in the calcs somewhere. It could also just be accounting for losses – 38mm is really not very much insulation and the space heating goes down by a couple of 000 kWh, so it does suggest its going into the hosue.

Do you have a stat and a timer on the cylinder? (See the note on ‘separate controls’).

Marianne

**Comment by TrystanLea** [comment]
> Do you have a stat and a timer on the cylinder? (See the note on ‘separate controls’).

Yes I do, Perhaps its the SAP calculation, the figures quoted here from another energy savings trust report do suggest considerably lower heat loss. I will dig into it a bit.
http://www.solarblogger.net/2012/11/heat-losses-from-hot-water-cylinders.html

**Comment by marianneURBED** [comment]
SAP makes all kind of allowances and 'worst case scenario' assumptions, because of its use as a regulatory compliance tool. It could be related to that. 

I'll go through it with Carlos when he's here this week and see if we can work out what's causing it.

**Comment by TrystanLea** [comment]
I had a check through the SAP document for the cylinder heat loss factor and found the issue. We are missing a 0 in the 38mm insulated cylinder values in MHEP.

![heatloss](https://user-images.githubusercontent.com/503186/37056276-d3f697b2-217b-11e8-9a75-e6a773ca4719.png)

So rather than 0.181 kWh/Litre/day it should be 0.0181 kWh/Litre/day

**Comment by marianneURBED** [comment]
Classic - a damn typo! Thanks for finding it.

________________________________
